ghost/i18n/locales/it/comments.json

```diff
@@ -35,18 +35,18 @@
     "Jamie Larson": "Andrea Rossi",
     "Join the discussion": "Partecipa alla discussione",
     "Just now": "Adesso",
-    "Load more ({{amount}})": "",
+    "Load more ({{amount}})": "Caricane altri",
```

Comment:
Sorry, I should have understood this from reading the rest of the translation. It's fixed now. I also removed the parentheses in the Italian string, since the “ne” at the end of “caricane” functions as a pronoun, equivalent to “Load {{amount}} more (of item).”